.github/CODEOWNERS

```diff
@@ -3,8 +3,8 @@
 
 *                                        @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
 /.github/workflows                       @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood @ijjk
-/packages/cli/src/commands/domains       @mglagola @anatrajkovska
-/packages/cli/src/commands/certs         @mglagola @anatrajkovska
+/packages/cli/src/commands/domains       @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
+/packages/cli/src/commands/certs         @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
 /packages/cli/src/commands/env           @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
```

Comment:
These are the same as `*` so I think we should just remove them:

```suggestion
```

Reply:
done!